docs/source/cuda/index.rst

```diff
@@ -7,6 +7,7 @@ Numba for CUDA GPUs
 .. toctree::
 
    overview.rst
+   types.rst
```

Comment:
I think it would be better to link to this in the reference guide, underneath the "CUDA Kernel API" section. IIRC this is where the content previously was when it was in the "Intrinsic Attributes and Functions" section.